### PR TITLE
chore: add @rclarkelinz to CODEOWNERS

### PR DESCRIPTION
**Pull request by dwsilk**
Automatically assign @rclarkelinz to review changes to the aerial.json and event.*.json tilesets

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,3 @@
-* @linz/proj-basemaps-dev 
+* @linz/proj-basemaps-dev
+config/tileset/aerial.json @linz/proj-basemaps-dev @rclarkelinz
```
Comment (blacha) [inline .github/CODEOWNERS:2]: do we need to double the devs here, I thought * would flow down to everything?

Comment (dwsilk) [reply .github/CODEOWNERS:2]: Docs examples says:

# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
# @global-owner1 and @global-owner2 will be requested for
# review when someone opens a pull request.
*       @global-owner1 @global-owner2

I don't want only @rclarkelinz tagged when the aerial.json changes.

Comment (dwsilk) [reply .github/CODEOWNERS:2]: Doesn't appear to flow down.

Comment (blacha) [reply .github/CODEOWNERS:2]: TIL! thanks

**Review by blacha — APPROVED** [review]

**Pull request merged**